README.md

```diff
@@ -9,3 +9,11 @@ repositories {
     ...
 }
 ```
+
+To be able publish you must set publish.url.snapshot=https://... and/or publish.url.releases=https://... in your gradle.properties
```

Comment:
Please make this more informative for novice users. It needs to be maven repos and just provide some real world examples